code/rtsp_save_process/rtsp_recorder.py

```
@@ -0,0 +1,706 @@
+        if current_pos >= total_frames - 1:
+            cap.set(cv2.CAP_PROP_POS_FRAMES, 0)
+            self.local_video_loop_count += 1
+            self.stats['loops_completed'] += 1
+            self.video_start_time = time.time()  # 시간 기준 재설정
+            
+            logger.info(f"[{self.source_name}] 영상 반복 재생 #{self.local_video_loop_count}")
+            return True
+        
+        return True
+        """새 세그먼트 생성"""
+        timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
```

Ruff (0.12.2)

Local variable `timestamp` is assigned to but never used

Remove assignment to unused variable `timestamp`

(F841)